### ISSUE TITLE
Hi, I find a problem?

### ISSUE THREAD

**Issue by BCWang93** [jvanvugt/pytorch-unet#5]
When I use this code to train voc2012,I found some error,the origin image size is
torch.Size([1, 3, 500, 375])#image
torch.Size([1, 500, 375])#label
but the output size of the net is 
torch.Size([1, 21, 308, 180])#net output
So,I think there have some problem?Do you know this is why?Thanks!

**Comment by jvanvugt** [comment]
By default, U-Net uses valid convolutions, which means that your feature map becomes smaller after every convolution. You can enable zero-padding to make sure your output has the same size as your input, just use `padding=True` when you create the model. One issue that will remain is that your image has an odd width (375), so you will lose a pixel in the maxpooling. The easiest fix is to pad your input with one column of zeros, so the width is 376. You probably want to pad a bit more even, because the image size has to be `depth - 1` times divisble by two.
